public/doc/openapi-3-v1.0.0.json

```diff
@@ -717,6 +717,66 @@
                 }
             }
         },
+        "/workflows/resolve": {
+            "post": {
+                "tags": [
+                    "Workflow"
+                ],
+                "summary": "Get a list of workflows ids",
+                "description": "", 
+                "operationId": "resolveWorkflows",
+                "requestBody": {
+                   "content": {
+                       "application/json": {
+                           "schema": {
+                               "type": "object",
+                               "properties": {
+                                   "tags": {
+                                       "type": "array",
+                                       "items": {
+                                           "$ref": "#/components/schemas/Tag"
```

Comment:
All new endpoint can share the same schema. Let's name it `#/components/schemas/ResourceObject'